config/routes.rb

```diff
@@ -4,5 +4,6 @@
 
   root "home#index"
 
-  get 'search', to: 'searches#index'
+  get '/search', to: 'searches#index'
+  post '/add', to: 'searches#add_to_favorites'
```

Comment:
This endpoint doesn't do any searching, so it doesn't make sense in a "SearchesController". PlacesController would be better.